cmd/pomerium/main.go

```diff
@@ -16,9 +17,10 @@ var (
 )
 
 func main() {
-	if err := run(context.Background()); err != nil {
+	if err := run(context.Background()); !errors.Is(err, context.Canceled) {
```

Comment:
I think this could be more sophisticated by only catching certain signals, but I'm not sure it's worth the complexity.